sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1057,6 +1057,25 @@ extended either for the lifetime of the graph (including both modifiable graphs
 and the executable graphs created from them) or until the buffer is no longer
 required by the graph (such as after being replaced through executable graph update).
 
+=== Buffer Limtations
+
+Because of the delayed execution of a recorded graph it is not possible to support
+code which relies on the copy-back on destruction behaviour of buffers.
+Typically applications would rely on this behaviour to do work on the host which
+cannot inherently be captured inside a command graph. Thus when recording to a graph
```

Comment:
"recording to a graph" - would it also be an error to create a graph node using the explicit API which uses an accessor that causes write-back? we should say so one way or another